view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -56,13 +56,112 @@ define(
                 return this;
             },
 
+            /**
+             * Get Installment minimum
+             * this function respects info from: https://www.omise.co/installment-payment
+             *
+             * NOTE: in the future this function should return data from capabilities object.
+             *
+             * @return {integer}
+             */
+            getInstallmentMinimum(id) {
+                switch (id) {
+                    case 'kbank':        return 500;
+                    case 'bbl':          return 500;
+                    case 'bay':          return 300;
+                    case 'first_choice': return 300;
+                    case 'ktc':          return 300;
+                    default:             return NaN;
+                }
+            },
+
+            /**
+             * Get Installment monthly interest rate
+             *
+             * NOTE: in the future this function should return data from capabilities object.
+             *
+             * @return {float}
+             */
+            getinstallmentInterestRate(id) {
+                switch (id) {
+                    case 'kbank':        return 0.0065;
+                    case 'bbl':          return 0.008;
+                    case 'bay':          return 0.008;
+                    case 'first_choice': return 0.013;
+                    case 'ktc':          return 0.008;
+                    default:             return NaN;
+                }
+            },
+
+            /**
+             * Get VAT tax rate
+             *
+             * NOTE: in the future this function should return data from capabilities object.
+             *
+             * @return {float}
+             */
+            getVatRate() {
+                return 0.07;
+            },
+
+            /**
+             * Get zero interests setting
+             *
+             * @return {string}
+             */
+            isZeroInterests() {
+                return window.checkoutConfig.is_zero_interest;
+            },
+
+            /**
+             * Calculates single installment amount
+             *
+             * @return {integer}
+             */
+            calculateSingleInstallmentAmount(id, terms) {
+                const total = this.getTotal();
+                const tax = this.getVatRate();
+
+                if (this.isZeroInterests()) {
+                    //merchant pays interests
+                    return (total / terms).toFixed(0)
+                }
+
+                // get yearly interests setting
+                const rate = this.getinstallmentInterestRate(id);
+                const interests = ((1 + tax) * rate * terms * total);
+
+                return + ( ( (total + interests ) / terms ).toFixed( 0 ) );
```

Comment:
What is `+` doing there?

Reply:
it is converting string to integer

Reply:
Again, should we be rounding to baht? (if this is baht and not subunits?)